src/coreclr/src/jit/bitsetasshortlong.h

```diff
@@ -412,12 +412,16 @@ class BitSetOps</*BitSetType*/ BitSetShortLongRep,
             char*    ptr             = res;
             if (sizeof(size_t) == sizeof(int64_t))
             {
+#ifdef HOST_64BIT
                 sprintf_s(ptr, remaining, "%016llX", bits);
```

Comment:
You can use `%016zX` here instead of differentiating between the 32 and 64 bit. And the same is true at other similar places. The `z` represents size_t.